### PR TITLE
PAPI needs contains placeholders

### PR DESCRIPTION
**Pull request by montlikadani**
This means that if a user uses PAPI in the server, but not in the VillageDefense plugin, so use it only when the message contains a placeholder.

### CODE REVIEW

**Comment by montlikadani** [comment]
Other:
- We don't need join & quit messages.
- Added [vd] abridgment for signs.

File: src/main/java/pl/plajer/villagedefense3/handlers/BungeeManager.java [hunk 1]
```diff
@@ -82,13 +82,11 @@ public void onServerListPing(ServerListPingEvent event) {
 
   @EventHandler(priority = EventPriority.HIGHEST)
   public void onJoin(final PlayerJoinEvent event) {
-    event.setJoinMessage("");
```
Comment (Plajer) [inline src/main/java/pl/plajer/villagedefense3/handlers/BungeeManager.java]: Why did you remove that, join and quit should be silent for bungee, you don't see messages "xyz joined the game" on big networks.

Comment (montlikadani) [reply src/main/java/pl/plajer/villagedefense3/handlers/BungeeManager.java]: This I think is superfluous unnecessary, as there are plugins that handle join & quit messages.

Comment (Plajer) [reply src/main/java/pl/plajer/villagedefense3/handlers/BungeeManager.java]: You think it's unnecessary while someone might see this feature as something good. Minigame is meant to run by its own without installing additional plugins.

Comment (Plajer) [inline src/main/java/pl/plajer/villagedefense3/handlers/BungeeManager.java]: Add code for setting quit messages and it will be okay.

Comment (montlikadani) [reply src/main/java/pl/plajer/villagedefense3/handlers/BungeeManager.java]: Rather, I leave this feature, may be another user.

**Comment by Plajer** [comment]
Thanks a lot for your contribution! We're appreciating that!
